examples/test.tfvars

```diff
@@ -32,3 +32,7 @@ grafana_resources = {
     memory_requests = "1Gi"
 }
 grafana_pv_size = "20"
+kube_state_metrics_resources = {
+    cpu_requests    = "2m"
+    memory_requests = "64Mi"
+}
```

Comment:
i'm not very into this, as we are basically going back piece by piece to the helm values, we should instead make it way simpler and offering just a way to inject a values file to let the user to configure all the aspects of the helm chart, without worrying about having it supported by the terraform module or not